modules/cert-manager-override-scheduling.adoc

```diff
@@ -0,0 +1,106 @@
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed version 1.15.0 or later of the {cert-manager-operator}.
+
+.Procedure
+
+* Update the `certmanager.operator` custom resource to configure pod scheduling overrides for the desired components. Use the `overrideScheduling` field under `controllerConfig`, `webhookConfig`, or `cainjectorConfig` to define `nodeSelector` and `tolerations` settings.
++
+.Example output
+[source,yaml]
+----
+$ oc patch certmanager.operator cluster --type=merge -p="
+spec:
+  controllerConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <1>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <2>
+  webhookConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <3>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <4>
+  cainjectorConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <5>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <6>
+----
+<1> Defines the `nodeSelector` for the cert-manager controller deployment.
+<2> Defines the `toleration` for the cert-manager controller deployment.
+<3> Defines the `nodeSelector` for the cert-manager webhook deployment.
+<4> Defines the `toleration` for the cert-manager webhook deployment.
+<5> Defines the `nodeSelector` for the cert-manager cainjector deployment.
+<6> Defines the `toleration` for the cert-manager cainjector deployment.
+
+
+.Verification
+
+. Verify pod scheduling settings for `cert-manager` pods:
+
+.. Check the deployments in the `cert-manager` namespace to confirm they have the correct `nodeSelector` and `tolerations`:
++
+[source,terminal]
+----
+$ oc get pods -n cert-manager -o wide
+----
++
+.Example output
+[source,yaml]
+----
+NAME                                       READY   STATUS    RESTARTS   AGE   IP            NODE                         NOMINATED NODE   READINESS GATES
+cert-manager-58d9c69db4-78mzp              1/1     Running   0          10m   10.129.0.36   ip-10-0-1-106.ec2.internal   <none>           <none>
+cert-manager-cainjector-85b6987c66-rhzf7   1/1     Running   0          11m   10.128.0.39   ip-10-0-1-136.ec2.internal   <none>           <none>
+cert-manager-webhook-7f54b4b858-29bsp      1/1     Running   0          11m   10.129.0.35   ip-10-0-1-106.ec2.internal   <none>           <none>
+----
+
+.. To check the `nodeSelector` and `tolerations` settings applied to deployments, run the following:
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps), "Use imperative mood in the first sentence of each step."
```suggestion
.. Check the `nodeSelector` and `tolerations` settings applied to deployments by running the following command:
```